packages/toolkit/src/view/pipeline/view-pipeline/ViewPipeline.tsx

```diff
@@ -48,6 +51,9 @@ export const ViewPipeline = () => {
     }
     setCurrentVersion(pipeline.data?.releases[0]?.id);
 
+    updatePipelineId(() => pipeline.data.id);
+    updatePipelineName(() => pipeline.data.name);
+
```

Comment:
@iamnamananand996 You can't do this kind of side-effect in a useMemo, that is an anti-pattern, please use another useEffect to operate this task